tor_ocr/main.py

```diff
@@ -136,8 +181,11 @@ def run(config):
 
     tor_post = config.r.submission(id=clean_id(tor_post_id))
 
-    thing_to_reply_to = tor_post.reply(_(base_comment))
-    for chunk in chunks(result, 9000):
+    thing_to_reply_to = tor_post.reply(
+        _(base_comment.format(result['process_time_in_ms'] / 1000))
```

Comment:
Are we doing any handling of `result['process_time_in_ms']` in case the datatype isn't an int? What if it's a string or `None`?

Reply:
I'll add a cast to an int when I make the dict, and add some handing if it fails.

Reply:
It should always return an int, because that's what the docs say 😜

Reply:
It's just generally a good policy to validate your input when coming from a source outside of your control. Did you create it? Nope? Validate it before you start working with it.

I'm sure they keep their docs up-to-date now, but I'd prefer an error indicating it can't cast a dict to an int than some strange behavior of python trying to work with disparate data types.

Reply:
Agreed! Also I added a cast in the dict definition in case your missed it 😛

Reply:
Oh I'm dumb, I was thinking of the exit code, when you meant the `process_time_in_ms`. My bad!

Also, GitHub this change has been fixed. Stop saying that it needs to be fixed.